PMMP 5.0.0

Yookou opened the pull request:
<!-- Failure to complete the required fields will result in the pull request being closed. -->
Please make sure your issue complies with these guidelines:
- * [ ] Use same formatting
- * [ ] Changes must have been tested on PMMP.
- * [ ] Unless it is a minor code modification, you must use an IDE.
- * [ ] Have a detailed title.

### What does the PR change?
<!-- 
Does your Pull Request:
- resolve a bug? If so, link the issue with the PR and add explain what caused the issue.
- enhance the plugin? If so, explain what this adds, including why it should be added.
-->

### Testing Environment
<!-- Use `/version` for PMMP version -->
* PocketMine-MP:
* PHP:
* Server OS:

<!--- Provide any extra information below  -->
### Extra Information

Yookou committed:
PMMP 5.0.0

Yookou committed:
Update README.md

nicholass003 commented on src/DaPigGuy/PiggyCrates/commands/CrateCommand.php:
```diff
@@ -14,7 +14,14 @@
 {
     /** @var PiggyCrates */
-    protected $plugin;
+    protected \pocketmine\plugin\Plugin $plugin;
```
I don't think it's that bad, for clean code it's better to use `use` statements.
```php
use pocketmine\plugin\Plugin;
```
```suggestion
    protected Plugin $plugin;
```

poggit-bot committed:
Create .poggit.yml
Poggit-CI is enabled for this repo by @Yakuuuuuuuuuuuuuuu
Visit the Poggit-CI page for this repo at https://poggit.pmmp.io/ci/Yakuuuuuuuuuuuuuuu/PiggyCrates
